spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/task/AsyncTaskExecutorProperties.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2012-2013 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.springframework.boot.autoconfigure.task;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ *
+ * Configuration properties for TaskExecutor
+ *
+ * @author Tiarê Balbi Bonamini
+ */
+@ConfigurationProperties(prefix = "spring.task")
+public class AsyncTaskExecutorProperties {
+
+	private String threadNamePrefix = "";
+
+	private int concurrencyLimit = -1;
+
+	public String getThreadNamePrefix() {
+		return threadNamePrefix;
+	}
+
+	public void setThreadNamePrefix(String threadNamePrefix) {
+		this.threadNamePrefix = threadNamePrefix;
+	}
+
+	/**
+	 * Set the maximum number of concurrent access attempts allowed. -1 indicates
+	 * unbounded concurrency.
+	 */
+	public int getConcurrencyLimit() {
```

Comment:
Please add meta-data description on the fields instead (as we use that for content assistance). Check other `XyzProperties` for a reference.